api/src/main/resources/marquez/db/migration/V52__alter_dataset_symlinks.sql

```diff
@@ -1,2 +1,3 @@
 /* SPDX-License-Identifier: Apache-2.0 */
+DROP VIEW IF EXISTS datasets_view;
```

Comment:
https://flywaydb.org/documentation/concepts/migrations.html#repeatable-migrations

> Repeatable migrations have a description and a checksum, but no version. Instead of being run just once, they are (re-)applied every time their checksum changes.

> Within a single migration run, repeatable migrations are always applied last, after all pending versioned migrations have been executed. Repeatable migrations are applied in the order of their description.

Reply:
As the doc says, `they are (re-)applied every time their checksum changes.` Just dropping the view will not create a view again and the application will break.